docs/admin/kubeadm.md

```diff
@@ -388,6 +388,15 @@ configuration file.
 
 If `KUBE_KUBERNETES_DIR` is specified, you may need to rewrite the arguments of the kubelet.
 (e.g. --kubeconfig, --pod-manifest-path)
+If `KUBE_REPO_PREFIX` is specified, you may need to set --pod-infra-container-image which is the pause image.
+```bash
```

Comment:
Put an extra newline before the bash commands so that there is some space around it. Thx